exercises/risiko/player.h

```diff
@@ -0,0 +1,91 @@
+#include<iostream>
+#include<cstdlib>
+#include<ctime>
+
+using namespace std;
+
+class risikoPlayer {
+    private:    
+        int N,M,O;
+        string name;
+        
+
+    public:
+        risikoPlayer(string name): name(name) {
+            // rolling dices
+            int arrOfDices[3];
+            
+            //srand(time(NULL));
+            for(int i=0; i<3; i++) {
+                arrOfDices[i]=rand()%(6)+1;
+            }
+
+            //first highest dice
+            int temp_max=arrOfDices[0];
+            int max_index;
+            for(int i=1; i<3; i++) {
+                if(temp_max<arrOfDices[i]) {
+                    temp_max=arrOfDices[i];
+                    max_index = i;
+                }
+            }
+
+            this->N=temp_max;
+
+            //second highest dice
+            int temp_second=0;
+            int second_index;
+            for(int i=0; i<3 && i!=max_index; i++) {
+                if(arrOfDices[i]>temp_second) {
+                    temp_second=arrOfDices[i];
+                    second_index=i;
+                }
+            }
+            
+            this->M=temp_second;
+
+            //third highest dice
+            int index=0;
+            while(index == max_index || index == second_index) 
+                index++;
+            
+            this->O=arrOfDices[index];
```

Comment:
`O` come attributo  è poco esplicativo, potresti dare dei nomi alle variabili che diano più informazioni, inoltre usa più spazi non scrivere tutto attaccato :P

Reply:
Per quanto riguarda il nome delle variabili, dovrei rinominare secondo te solo gli attributi della classe oppure in generale un refactor sulle variabili?